mindmeld/components/classifier.py

```diff
@@ -403,6 +403,8 @@ def _get_model_config(loaded_config=None, **kwargs):
         Returns:
             ModelConfig: The model configuration corresponding to the provided config name
         """
+        if 'params' in loaded_config and 'params' in kwargs:
+            kwargs['params'].update(loaded_config['params'])
```

Comment:
This feels backwards. We're overwriting the passed in params with the existing config? 

I think that second line should just be something like:
`merged = {**loaded_config['params'], **kwargs['params']}`

Reply:
Nice catch! 
![gifntext-gif](https://user-images.githubusercontent.com/16606656/181075731-742f74e8-0798-4daf-ac4e-7574bce7a6ea.gif)
** BLOCKED BY JAMES **